protocol/app/app.go

```diff
@@ -509,6 +509,7 @@ func New(
 		grpc.NewServer(),
 		&lib.FileHandlerImpl{},
 		daemonFlags.Shared.SocketAddress,
+		daemonFlags.Shared.GrpcServerAddress,
```

Comment:
Small nit: can you give this daemon a custom name rather than the gRPC server address? Maybe `app-grpc-server`?

Reply:
That's not a daemon. It's literally the grpc address that the daemons dial into to find the cosmos query server.